Sources/TuistGraph/Models/ProjectOptions.swift

```diff
@@ -6,6 +6,9 @@ extension Project {
         /// Defines how to generate automatic schemes
         public let automaticSchemesOptions: AutomaticSchemesOptions
 
+        /// Configures the development region.
+        public let developmentRegion: String?
+
```

Comment:
Where does this came from?

Reply:
This change is not related to the fix at all. ProjectDescription.Project.Options support this property and when converting these types to TuistGraph.Project.Options it was missing. So, I added it just for consistency.

Reply:
if it's not here it means it's mapped in some other way, and is not needed here :D

Reply:
🤦‍♂️  sure I will remove those :D